torch/_inductor/codegen/cpp.py

```diff
@@ -372,6 +372,10 @@ def _print_Min(self, expr):
             il = "{" + ", ".join(args) + "}"
             return f"std::min({il})"
 
+    def _print_align(self, expr):
```

Comment:
Did you mean to override this?

Reply:
This isn't an override. We have defined `align` as a custom `sympy.Function`, so we need to teach the printer how to print this new sympy expression.

Reply:
Oh, I thought you added `_print_align` to the base class in `common.py`.

Reply:
Ohh. I thought the base `ExprPrinter` class was in a sympy library; I see now that it is defined by us. I'll move the two definitions of `_print_align` into the base class.